Index.html

```diff
@@ -8,60 +8,63 @@
     <title>Renome restaurant</title>
   </head>
 
-  <main>
-    <body>
-      <div class="navigation">
-        <p class="navigation-left">Renome</p>
+  <body>
+    <header>
+      <nav class="navigation">
+        <p class="navigation-logo">Renome</p>
         <div class="navigation-right">
           <img
-            class="navigation-right-cart"
+            class="navigation-right_cart"
             src="/images/cart-shopping-solid.svg"
-            alt="cart-shopping-solid"
+            alt="shopping cart icon"
           />
-          <p class="navigation-count">2</p>
+          <p class="navigation-right_count">2</p>
           <img
-            class="navigation-divide"
+            class="navigation-right_divide"
             src="/images/9034432_divider_vertical_icon.svg"
+            alt="divider icon"
           />
           <img
-            class="naviagtion-right-burger"
+            class="naviagtion-right_burger"
             src="/images/6351905_burger_checklist_list_menu_navigation_icon.svg"
-            alt="burger menu"
+            alt="burger menu icon"
           />
         </div>
-      </div>
+      </nav>
+    </header>
 
-      <div class="carousel-container">
+    <main>
+      <div class="carousel">
         <div class="carousel-images">
           <img
             src="/images/carousel_img_1.png"
             alt="peper salt timian and knife"
           />
-          <div class="carousel-container-text">
-            <h4 class="carousel-title">made with love</h4>
-            <p class="carousel-subtitle">for you</p>
-            <button class="left"><a class="arrow back">&#8249;</a></button>
-            <button class="right"><a class="arrow back">&#8250;</a></button>
-          </div>
+        </div>
+        <div class="carousel-text">
+          <h4 class="carousel-title">made with love</h4>
+          <p class="carousel-subtitle">for you</p>
+          <button class="carousel-arrow">
+            <img src="/images/left_arrow_icon.svg" alt="arrow left" />
+          </button>
+          <button class="carousel-arrow">
+            <img src="/images/right_arrow_icon.svg" alt="arrow right" />
+          </button>
         </div>
       </div>
 
       <article>
         <div class="about">
-          <div class="about-image-container">
-            <div class="about-image-grid">
-              <div class="about-back-image">
-                <img
-                  src="/images/about_img_2.png"
-                  alt="white house from afar"
-                />
-              </div>
-              <div class="about-top-image">
-                <img
-                  src="/images/about_img_1.png"
-                  alt="close-up second house floor window"
-                />
-              </div>
+          <div class="about-image-grid">
+            <div class="about-image-grid_back">
+              <img src="/images/about_img_2.png"
+              alt="white house from afar" />
```

Comment:
The same here with attributes, better then have them in one line, if it's not too long

Reply:
I was thinking that it's better to keep them in separate lines, to be consistent through all code.  And in my opinion it doesn't fit in one line.